models/Import/Service/SendCalculatedResultService.php

```diff
@@ -67,7 +67,12 @@ public function sendByDeliveryExecutionId(string $deliveryExecutionId): array
 
         $isFullyGraded = $this->checkIsFullyGraded($deliveryExecutionId, $outcomeVariables);
 
-        $timestamp = DateHelper::formatMicrotime($this->getScoreTotalTimestamp($outcomeVariables));
+        $microtime = $deliveryExecution->getFinishTime();
```

Comment:
Can we call this $finishMicroTime? $microtime is not very meaningful.

Reply:
Couple lines below it takes microtime from latest outcome so its not $finishMicrotime.